AppCenterCrashes/AppCenterCrashes/MSCrashes.mm

```diff
@@ -671,10 +671,10 @@ - (void)configureCrashReporterWithUncaughtExceptionHandlerEnabled:(BOOL)enableUn
   }
 #endif
   PLCrashReporterSymbolicationStrategy symbolicationStrategy = PLCrashReporterSymbolicationStrategyNone;
-  MSPLCrashReporterConfig *config = [[MSPLCrashReporterConfig alloc] initWithSignalHandlerType:signalHandlerType
+  PLCrashReporterConfig *config = [[PLCrashReporterConfig alloc] initWithSignalHandlerType:signalHandlerType
                                                                          symbolicationStrategy:symbolicationStrategy
                                                         shouldRegisterUncaughtExceptionHandler:enableUncaughtExceptionHandler];
-  self.plCrashReporter = [[MSPLCrashReporter alloc] initWithConfiguration:config];
+  self.plCrashReporter = [[PLCrashReporter alloc] initWithConfiguration:config];
```

Comment:
I synced with @jaeklim about this, let's have a comment somewhere in the code, possibly above the PLCR init, where we explain the prefixing and why we don't need it in the SDK source code. This will avoid future confusion.